python/nav/web/seeddb/page/management_profile/forms.py

```diff
@@ -90,6 +90,49 @@ class Meta(object):
     )
 
 
+class NapalmForm(ProtocolSpecificMixIn, forms.ModelForm):
+    PROTOCOL = ManagementProfile.PROTOCOL_NAPALM
+    PROTOCOL_NAME = PROTOCOL_CHOICES.get(PROTOCOL)
+
+    class Meta(object):
```

Comment:
Why is Meta before the field definitions?

Reply:
Because... that's what was used for all the other forms in the same file, and no one complained on those pull requests?